src/domain/comment.js

```diff
@@ -0,0 +1,47 @@
+import dbClient from '../utils/dbClient.js'
+
+const getAllCommentsDb = async () => {
+  return await dbClient.comment.findMany()
+}
+
+const postFound = async (postId) => {
```

Comment:
This suggests it's a variable that contains a boolean value. `findPostById` would be a better name. Additionally, this function shouldn't be in the `comment` domain, but in the `post` domain